mxnet-bot/LabelBotAddLabels/LabelBot.py

```diff
@@ -0,0 +1,144 @@
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+import json
+import os
+from botocore.vendored import requests
+import re
+import logging
+import secret_manager
+logging.basicConfig(level=logging.INFO)
+
+class LabelBot:
+    secret = json.loads(secret_manager.get_secret())
```

Comment:
This looks much better.

Can we add GITHUB_USER, GITHUB_OUATH_TOKEN and REPO as a constructor arguments so the class is more modular and testable? 
Also observe the python coding standards  (PEP8) for the REPO variables etc.

You can read the environment in the lambda handler when you create the class. After this change we are ready to go and merge.

Reply:
Done. Add github_use, github_oauth_token and repo as constructor arguments. Also set instance variables in lower_snake_cases.